docs/native/src/testing/testcases.rst

```diff
@@ -1,3 +1,163 @@
+
+#. Activate control to open in Office
+
+**RESULT: User should be sent to app store page for the corresponding app.**
+
+#. Install the Office app.
+#. After installation, go back to the Company app and activate the control to open in Office again.
+
+**RESULT: Office should start. User should be prompted for credentials to Company Service.**
+
+#. Enter credentials for company service.
+
+**RESULT: File should open in read only mode.**
+
+#. Click :guilabel:`Sign In` and sign in with a free Microsoft Account.
+
+**RESULT: File should open in edit mode if user is a consumer and read-only mode if the user is a commercial user.**
+
+#. Make changes [you will need to sign in with a subscription account for testing commercial user]
+#. Click :guilabel:`Back (<-)`
+#. Click :guilabel:`Open`
+
+**RESULT: Confirm company service is shown as a place.**
+
+2 - Open from Office - Fresh Install
+------------------------------------
+*This test verifies the flow of using Company Service for the first time from Office.*
+
+#. Launch a fresh install of Office.
+#. Go through the First Run Experience.
+#. Skip :guilabel:`Sign In.`
+#. Go to :menuselection:`Open --> Add a Place`
+
+**RESULT: Company service shows up. Verify the name and icon of your service.**
+
+#. Select your Company Service.
+#. Enter credentials.
+
+**RESULT: Root folder should show.**
+
+#. Browse around the folder structure in your service.
+
+**RESULT: Browse works as expected.**
+
+#. Open a file from :guilabel:`Browse`.
+
+**RESULT: File should open in read-only mode.**
+
+#. Click :guilabel:`Sign In` and sign in with a free Microsoft Account.
+
+**RESULT: File should open in edit mode if user is a consumer and read-only mode if the user is a commercial user.**
+
+#. Make changes [you will need to sign in with a subscription account for testing commercial user]
+#. Click :guilabel:`Back (<-)`
+#. Click "Open"
+
+**RESULT: File should have the previously saved changes. Ensure changes are being saved on Company service.**
+
+
+3 - Open from Company App - Repeat Usage
+----------------------------------------
+Repeat test 1 except with company service already added (i.e. from previous usage).
+
+
+4 - Open from Office - Repeat Usage
+-----------------------------------
+Repeat test 2 except with company service already added (i.e. from previous usage).
+
+
+5 - Save As [duplicate]
+-----------------------
+Verify ability to duplicate to Company Service, both by adding a new place and using an existing place.
+
+..  figure:: ../images/test_dupe.png  
+    :alt: A screenshot that shows the document actions in |Office iOS|. 
+
+    :guilabel: A screenshot that shows the document actions in |Office iOS|. 
```

Comment:
Don't use `guilabel` here. Only use it when you're referring to a specific button or something in the UI.